src/Raven.Studio/typescript/viewmodels/database/settings/studioConfiguration.ts

```diff
@@ -37,6 +42,16 @@ class studioConfiguration extends viewModelBase {
             .execute()
             .always(() => this.spinners.save(false));
     }
+
+    installApp() {
```

Comment:
which has to moved to server wide studio configuration. Current it is in db specific studio configuration.

Reply:
You want this moved to the server-wide Studio Configuration? OK.